### ISSUE TITLE
HTMLEditorControl - HyperLInk function any ideas?

### ISSUE THREAD

**Issue by kipcarter** [DevExpress/AjaxControlToolkit#97]
IE 11 the control appears t work just perfectly but I notice that there is one vexing issue with it.  After you have placed a URL behind text it just sits there unusable short of being copied and pasted into some other app because from IE 11 there doesn't appear to be a way to activate the hyperlink and go to where the link is actually pointing.  Anyone else seen this with another browser or also on 11. Anyone have a solution to this issue or a key combo to use the link correctly while clicking? I noted the same behavior with Chrome as well.  Now this was all prior to deployment so it was while in debug so it could also just be a symptom of the debugger as well.  Just would like some additional light on this because if the hyperlink function in the HTMLEditorControl doesn't work then it would be pointless for me to use the extension on this particular field where I have  it placed. 
Regards,
Kip...

**Comment by MikhailTymchukDX** [comment]
You need to switch to the preview mode, because in edit mode you can not click a link:
![image](https://cloud.githubusercontent.com/assets/17080984/15152814/62fd13fe-16df-11e6-8f81-2b89e7e89642.png)

**Comment by kipcarter** [comment]
Where do I find this 'preview mode' icon to click? I've looked around and not seeing it anywhere!
Thanks,
Kip...

**Comment by jafin** [comment]
@kipcarter 
For the HtmlEditorExtender set the Attribute DisplaySourceTab="True" To show the Preview toolbar.  For the HtmlEditor I think it is the same attribute.

**Comment by MikhailTymchukDX** [comment]
@jafin AjaxControlToolkit.HtmlEditor.Editor does not have such an attribute.

@kipcarter These icons should be in the bottom-left corner of the editor. If you cannot find it, please provide a sample project that illustrates this issue.

Please take into account that the Editor control was deprecated in the context of this thread: #70. 
We recommend using [HtmlEditorExtender](https://github.com/DevExpress/AjaxControlToolkit/wiki/HtmlEditor) instead.

**Comment by madhav5589** [comment]
@jafin 
I am using HtmlEditorExtender and I set DisplaySourceTab="True". I don't see Preview mode (magnifier) on the bottom toolbar. I see two button as shown in the demo [here](http://www.ajaxcontroltoolkit.net/HTMLEditorExtender/HTMLEditorExtender.aspx). 

@MikhailTymchukDX  Can you please guide for this ?

**Comment by jafin** [comment]
@madhav5589 sorry I misread your question, ignore my comment about sourcetab.

**Comment by madhav5589** [comment]
@MikhailTymchukDX  @jafin  I would like to know if I can have the same Preview button available in HtmlEditorExtender. As HtmlEditor is going to be deprecated, can we have the "Preview" button/functionality in HtmlEditorExtender ?

**Comment by MikhailTymchukDX** [comment]
@madhav5589 No, there is no Preview mode in HtmlEditorExtender.

I think this would be a useful feature for testing links, so we'll consider moving of this feature to HtmlEditorExtender.
Thank you for posting this idea!

**Comment by madhav5589** [comment]
@MikhailTymchukDX  Yes, that's good idea. Can you create a new feature request link or task id for tracking purpose ?

**Comment by MikhailTymchukDX** [comment]
@madhav5589 Sure, here it is: #128
